File: sdk/typescript/examples/00_helpers.mjs
// 示例共享:读取环境变量并创建客户端。
import { Combee } from "../dist/index.js";

export function client() {
  return new Combee({
    baseUrl: process.env.COMBEE_URL ?? "http://127.0.0.1:8080",
    apiKey: process.env.COMBEE_API_KEY ?? "dev-key",
  });
}

File: sdk/python/examples/00_helpers.py
"""示例共享:读取环境变量创建客户端。"""
import os
from combee import Combee


def client() -> Combee:
    return Combee(
        base_url=os.environ.get("COMBEE_URL", "http://127.0.0.1:8080"),
        api_key=os.environ.get("COMBEE_API_KEY", "dev-key"),
    )
